lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -882,6 +882,18 @@ MITIGATION_KBV_FAIL_M2B:
         ticfCriBeta:
           targetState: CRI_TICF_BEFORE_ANOTHER_WAY
 
+PYI_ESCAPE_M2B:
+  response:
+    type: page
+    pageId: pyi-escape-m2b
+  events:
+    next:
+      targetState: MITIGATION_01_IDENTITY_START_PAGE
```

Comment:
For my own clarity more than anything:

Why are we sending users to the Mitigation journey here? It feels weird to me because they haven't necessarily "failed" any CRI so what would there be to mitigate?

Reply:
Ah sorry I used the wrong one here, will change that